community/blog/2021-12 2021, the year that Arcus matured/README.md

```diff
@@ -0,0 +1,173 @@
+
+The changes that were made this year on this, are:
+* Capability to add operation parent ID to correlation information
+* Capability to add dependency name to dependency telemetry
+
+Besides those changes, we are working hard to make this library production-ready. It's a popular and very important library to have mature telemetry in your project.
+* Capability to track Azure Key Vault dependencies
+* Capability to use telemetry type filter outside Arcus-related Serilog sink
+* Capability to add public exception properties to exception telemetry
+
+See our [features docs](https://observability.arcus-azure.net/) for more information on our available functionality.
+
+## Templates
+The [templates repository](https://github.com/arcus-azure/arcus.templates) is probably the easiest place to get started with Arcus. It provides development-ready project templates for you to use at the start of your project. It already contains all the necessary best-practices and correctly use of Arcus libraries so you don't have to worry about infrastructure code but can focus on the business end.
+
+2021 was the year that we took this to another level and created project templates for Azure Functions.
+* Capability to create Azure Functions project with a HTTP endpoint trigger 
+* Capability to create Azure Functions project running a timer to log finished Databricks log runs
+
+There are already several project options and additional built-in functionality available for these new and existing project templates, so to make them more development-ready:
+* Capability to opt-in for health checks
+* Capability to track only 5xx requests in health endpoint
+* Capability to configure minimum Serilog logging level from application settings
+* Capability to use .NET project templates directly from Visual Studio 
+
+When the [messaging library](https://github.com/arcus-azure/arcus.messaging) is ready to provide the message routing outside message pumps, we create additional Azure Functions project templates with Azure Service Bus triggers. 
+See our [feature docs](https://templates.arcus-azure.net/) for more information on our available functionality.
+
+## Background Jobs
+Last but not least, we also have a new scheduled job in the [background jobs repository](https://github.com/arcus-azure/arcus.backgroundjobs). We already had a job that would notify us when an Azure Key Vault secret was expired, but now we also are able to notify consumers when a Service Principal client secrets is about to expire or is expired.
```

Comment:
Well, last but not least, but this is not the last topic  that is being described :)

Reply:
Right, removed that. thx